pages/api/sum.ts

```diff
@@ -0,0 +1,9 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { sum } from "@/pages/utils/sum";
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
+  const { num1, num2 } = req.query;
+  const result = sum(num1, num2);
+
+  res.status(200).json({ total: result });
```

Comment:
Hey @dorelljames. Although my understanding of Typescript may not be very strong, I have made an effort to improve its readability of this. I would greatly appreciate it if you could update it to this version.

```javascript
const total = sum(num1, num2);

res.status(200).json({ total });
```

Reply:
Yes, I like your idea. Let's do that.